### PR TITLE
Reduced 10 times the round sleep time in integration tests

### PR DESCRIPTION
**Pull request by raduchis**
optimizations of the sleep times in integration tests

### CODE REVIEW

File: integrationTests/testInitializer.go [hunk 1]
```diff
@@ -63,10 +63,10 @@ import (
 )
 
 // StepDelay is used so that transactions can disseminate properly
-var StepDelay = time.Second
+var StepDelay = time.Second / 10
 
 // SyncDelay is used so that nodes have enough time to sync
-var SyncDelay = time.Second * 2
+var SyncDelay = time.Second * 2 / 10
```
Comment (SebastianMarian) [inline integrationTests/testInitializer.go:69]: / 5 instead * 2 / 10

Comment (sasurobert) [reply integrationTests/testInitializer.go:69]: +1

**Review by iulianpascalau — COMMENTED** [review]
wondering why this works now and didn't worked in the past. GJ :+1:

File: integrationTests/testInitializer.go (hunk 1 above)
Comment (raduchis) [reply integrationTests/testInitializer.go:69]: fixed

**Review by SebastianMarian — DISMISSED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**